src/protocol/block.ml

```diff
@@ -10,12 +10,13 @@ type t = {
   previous_hash : BLAKE2B.t;
   author : Key_hash.t;
   block_height : int64;
+  consensus_round : int;
   operations : Protocol_operation.t list;
 }
 [@@deriving yojson]
 let hash, verify =
   let apply f ~state_root_hash ~withdrawal_handles_hash ~validators_hash
-      ~previous_hash ~author ~block_height ~operations =
+      ~previous_hash ~author ~block_height ~consensus_round ~operations =
     let to_yojson =
       [%to_yojson:
         BLAKE2B.t
```

Comment:
Excited for when we switch to Bin_prot and we can just serialize record types directly

Reply:
Actually, is this even the best we can do now? Why not use a JSON object or something? Large tuples are a little unwieldy right, since it's somewhat unclear without names/labels what each value is supposed to be?